examples/quickstart/notebooks/quickstart.ipynb

```diff
@@ -657,17 +652,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!zenml model-registry models list\n",
+    "from zenml.integrations.mlflow.steps.mlflow_deployer import (\n",
+    "    mlflow_model_registry_deployer_step,\n",
+    ")\n",
     "\n",
-    "!zenml model-registry models list-versions zenml-quickstart-model"
+    "model_deployer = mlflow_model_registry_deployer_step.with_options(\n",
+    "    parameters=dict(\n",
+    "        registry_model_name=model_name,\n",
+    "        registry_model_version=most_recent_model_version_number,\n",
+    "    )\n",
+    ")"
```

Comment:
Same here, maybe more intuitive to pass the params directly in the pipeline definition instead of using the `with_options` syntax